cypress/integration/basic.test.ts

```diff
@@ -45,4 +45,17 @@ context("Test the Hurricane Model app", () => {
       });
     });
   });
+
+  it("colorblind friendly SST", () => {
+    cy.get(".map-tab--mapTabImage--__hurr-v1__.map-tab--impactMaps--__hurr-v1__")
+      .click()
+      .then(() => {
+        cy.get(".sst-key--checkbox--__hurr-v1__ .MuiSvgIcon-root")
+          .click()
+          .then(() => {
+            cy.get('[src="dd7d513dc937501a59bcfe0b12d93fe3.png"]').should("be.visible");
```

Comment:
This filename is auto-generated by webpack loader. I guess it's based on the image content. I think we should try to avoid using them in general. They change each time we update the image.

Reply:
@pjanik , I think it is going to be hard to check for anything else because as far as I can tell, this image is the only thing that changes in the dom when the accessibility key is used. Kindly suggest something else that can be used for verifying that the map is using it.

Reply:
I see. In this case, let's leave it that way. If we update the asset, it'll be easy to update the test or just come up with another approach.